ray-on-gke/tpu/kuberay-tpu-webhook/deployments/deployment.yaml

```diff
@@ -3,6 +3,28 @@ kind: Namespace
 metadata:
   name: ray-system
 ---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pod-reader
+rules:
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: pod-reader
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
Generally try to avoid using the `default` service account, what do you think about creating a dedicated one for TPU webhook?

Reply:
That sounds good to me, I added one in [6f1fd5d](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/6f1fd5d7c85c5b9860d077a8e233feb825b05edd).